From 1b388bbcafe3f4d078f2afc1b50ef2f21638ae43 Mon Sep 17 00:00:00 2001
From: Dan Seefeldt <Dan.Seefeldt@microsoft.com>
Date: Thu, 1 Mar 2018 17:18:54 -0600
Subject: [PATCH] Update appInsights, common and vstest to place output
 correctly. (#329)

* Update appInsights, common and vstest to place output correctly.

* Change common and vstest projects to not have RepoAPI implemented per PR feedback
---
 repos/application-insights.proj | 2 +-
 repos/common.proj               | 1 +
 repos/vstest.proj               | 2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/repos/application-insights.proj b/repos/application-insights.proj
index e359104..28f74af 100644
--- a/repos/application-insights.proj
+++ b/repos/application-insights.proj
@@ -2,7 +2,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <PackagesOutput>$(ProjectDirectory)/bin/$(Configuration)</PackagesOutput>
+    <PackagesOutput>$(SubmoduleDirectory)/bin/$(Configuration)</PackagesOutput>
     <RepoApiImplemented>false</RepoApiImplemented>
   </PropertyGroup>
 
diff --git a/repos/common.proj b/repos/common.proj
index 6c3f7f1..81f6206 100644
--- a/repos/common.proj
+++ b/repos/common.proj
@@ -9,6 +9,7 @@
     <BuildArguments>/p:Configuration=$(Configuration)</BuildArguments>
     <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -NoTest $(BuildArguments)</BuildCommand>
     <CleanCommand>$(ProjectDirectory)/clean$(ShellExtension) $(BuildArguments)</CleanCommand>
+    <RepoApiImplemented>false</RepoApiImplemented>
   </PropertyGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))/dir.targets" />
diff --git a/repos/vstest.proj b/repos/vstest.proj
index 544cab7..fefa76f 100644
--- a/repos/vstest.proj
+++ b/repos/vstest.proj
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -DotnetBuildFromSource -DotnetCoreSdkDir $(DotNetCliToolDir) -c $(Configuration) -r $(TargetRid) -v 15.3.0 -vs preview-20170628-02</BuildCommand>
     <PackagesOutput>$(ProjectDirectory)/artifacts/$(Configuration)/packages</PackagesOutput>
-    <RepoApiImplemented>true</RepoApiImplemented>
+    <RepoApiImplemented>false</RepoApiImplemented>
   </PropertyGroup>
 
   <ItemGroup>
-- 
2.7.4

